src/Benchotron/StdIO.purs

```diff
@@ -19,14 +21,15 @@ foreign import stderrWrite ::
 
 question :: forall e.
   String ->
-  (String -> Eff (console :: CONSOLE | e) Unit) ->
-  Eff (console :: CONSOLE | e) Unit
+  (String -> 
+   Eff (readline::READLINE, console :: CONSOLE, err::EXCEPTION | e) Unit) ->
```

Comment:
Sorry, I didn't notice this before - can we keep the whitespace consistent here please? That is, one space either side of `::`.